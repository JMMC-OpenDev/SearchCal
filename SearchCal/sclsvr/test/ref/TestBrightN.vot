version https://git-lfs.github.com/spec/v1
oid sha256:f46ff6c02370bcfef4cb81daa1c7925d937fd60de2303bb9baaa0645e9b7ff1e
size 136349
