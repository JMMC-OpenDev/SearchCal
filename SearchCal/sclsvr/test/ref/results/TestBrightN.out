version https://git-lfs.github.com/spec/v1
oid sha256:ff900e975e5ca09dac29b43f39c4fe67adcd15722317cd143ab8ab8647673d79
size 23174
